DfE.FindInformationAcademiesTrusts.Data.AcademiesDb/Factories/GovernorFactory.cs

```diff
@@ -0,0 +1,39 @@
+using DfE.FindInformationAcademiesTrusts.Data.AcademiesDb.Extensions;
+using DfE.FindInformationAcademiesTrusts.Data.AcademiesDb.Models;
+
+namespace DfE.FindInformationAcademiesTrusts.Data.AcademiesDb.Factories;
+
+public interface IGovernorFactory
+{
+    Governor CreateFrom(Governance g);
```

Comment:
Could this parameter be named governance? (should also remove the code smell?)

Reply:
Will update in next PR where I'm planning to rename the 'Governance' type to 'GiasGovernance' (it was so confusing when building this!)